README.md

```diff
@@ -248,11 +250,84 @@ functionalities, etc.
 
 An example is worth a thousand words. Check out our Hello World demo (available
 to you in different flavors):
-- [Hello World (AMD + bower)](examples/amd-bower/).
-- [Hello World (Node.js + npm)](examples/node-npm/).
-- [Hello World (plain JavaScript)](examples/plain-javascript/).
+- [Hello World (AMD + bower)](examples/hello-world-amd-bower/).
+- [Hello World (Node.js + npm)](examples/hello-world-node-npm/).
+- [Hello World (plain JavaScript)](examples/hello-world-plain-javascript/).
 
 
+### Performance
+
+When formatting or parsing, there's actually a two step process: (a) the
+formatter (or parser) creation and (b) its execution, where creation takes
+considerably more time (more expensive) than execution. The difference is an
```

Comment:
Like "where creation takes an order of magnitude more time (more expensive) than execution"? Just double checking.